packages/ti_threatconnect/elasticsearch/transform/latest/transform.yml

```
@@ -10,7 +10,7 @@ source:
 # time field type conflicts.
 dest:
   index: "logs-ti_threatconnect_latest.dest_indicator-8"
-  pipeline: "1.10.2-tactics_compatibility"
+  pipeline: "1.10.3-tactics_compatibility"
```

```suggestion
  pipeline: "{{ ingestPipelineName "tactics_compatibility" }}"
```

See https://github.com/elastic/integrations/pull/16175

---

Update in [commit](https://github.com/elastic/integrations/pull/16187/commits/e37be207b46a0ebd4e3518238190d49717e93b51).